addons/actions/README.md

```diff
@@ -129,18 +129,20 @@ action('my-action', {
 |`clearOnStoryChange`|Boolean|Flag whether to clear the action logger when switching away from the current story.|`true`|
 |`limit`|Number|Limits the number of items logged in the action logger|`50`|
 
-## withActions decorator
+## Declarative Configuration via Parameters
 
-You can define action handles in a declarative way using `withActions` decorators. It accepts the same arguments as [`actions`](#multiple-actions)
```

Comment:
I don't really understand this line, it doesn't seem correct ("It accepts the same arguments as [`actions`]"). I realise this was here before but it doesn't seem right at all. The arguments are very much different.